hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/TypeInfoLogicalTypeVisitor.java

```diff
@@ -144,6 +145,16 @@ public TypeInfo visit(TimestampType timestampType) {
     }
   }
 
+  @Override
+  public TypeInfo visit(LocalZonedTimestampType localZonedTimestampType) {
+    int precision = localZonedTimestampType.getPrecision();
+    if (precision >= 0 && precision <= 9) {
+      return TypeInfoFactory.timestampTypeInfo;
+    } else {
```

Comment:
Can we add a test case in `TestHoodieHiveCatalog`.

Reply:
ok i will add